### ISSUE TITLE
Hotkeys Popup Text Hard to See in Dark Mode

### ISSUE THREAD

**Issue by sbelsk** [Submitty/Submitty#6512]
**Describe the bug**
When grading a lab (checkpoint) gradeable, and clicking the Settings/Hotkeys button, the text in the popup is hard to read in dark mode.
![dark_theme](https://user-images.githubusercontent.com/71195502/119662291-38ab6980-bdff-11eb-8c08-948fd0509c6d.png)

**Expected behavior**
The text should turn white or black accordingly when the website theme is changed.

**Comment by sbelsk** [comment]
I looked into this and tried the following things in attempt to change the table style and row colors:
- Adding bootstrap classes to the tables
- Adding custom CSS classes trying to override the existing styles
- As a last resort -- adding inline style to the table elements

My goal was to make it look more like the table below, with alternating dark-colored rows. My attempts did not work; I only managed to change the dark-colored rows and could not change the rows which are colored with --standard-pale-gray, which are the ones that make the text hard to read in Dark Mode. 
![image](https://user-images.githubusercontent.com/71195502/119677148-7793ec00-be0c-11eb-9b21-34784aa568a6.png)

I probably missed something, but if anyone wants to take this, please do :)
